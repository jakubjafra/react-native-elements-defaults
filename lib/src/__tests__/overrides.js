const overwrites = require('../overwrites');

describe('overwrites', () => {
    it('should contain all components', () => {
        expect(Object.keys(overwrites)).toMatchSnapshot();
    });
});
