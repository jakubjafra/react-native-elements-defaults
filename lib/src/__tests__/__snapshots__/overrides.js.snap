// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`overwrites should contain all components 1`] = `
Array [
  "Avatar",
  "Badge",
  "Button",
  "ButtonGroup",
  "Card",
  "CheckBox",
  "$colors",
  "Divider",
  "FeaturedTile",
  "$fonts",
  "FormInput",
  "FormLabel",
  "FormValidationMessage",
  "Header",
  "Icon",
  "List",
  "ListItem",
  "PricingCard",
  "Rating",
  "Search",
  "Slider",
  "SocialIcon",
  "Text",
  "Tile",
]
`;
